plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/core-profiler.spec.js

```diff
@@ -16,7 +16,7 @@ test.describe( 'Store owner can complete the core profiler', () => {
 			).toBeVisible();
 			await page
 				.getByRole( 'checkbox', { name: 'I agree to share my data' } )
-				.check();
+				.uncheck();
```

Comment:
The font-picker has different outcomes based on whether tracking is enabled or not.  So this was causing an unpredictable dependency between the tests.